webapp/src/components/modals/create_issue/__snapshots__/create_issue.test.jsx.snap

```diff
@@ -57,7 +57,6 @@ exports[`CreateIssueModal should render correctly with default props 1`] = `
           value={null}
         />
         <Input
-          disabled={false}
```

Comment:
I'm surprised to see that the snap was not up to date. Do we need CI check to ensure that?

Reply:
I don't think we need a CI check for the same, the web app test cases will fail if the snapshot is not updated. I wonder how this was not already on master